src/process-tasks.ts

```diff
@@ -23,15 +24,21 @@ export async function processTasks(
   >,
 ): Promise<[boolean, Data]> {
   const {
-    octokit,
+    ghToken,
     ghUser: username,
     dataFile,
     dataTasks,
     taskSkip,
     taskName,
     taskParams,
   } = ctx
+
+  if (!ghToken) throw new Error('GitHub token is required for data gathering')
+  if (!username)
```

Comment:
We can use without gh username:
```
node dist/src/main.js --data data/antonmedv.json
```
As data/antonmedv.json already contains username.